test/aws/centos_repos.yml

```diff
@@ -0,0 +1,52 @@
+---
+# Note: OpenShift repositories under CentOS may be shipped through the
+# "centos-release-openshift-origin" package which configures the repository.
+# This task matches the file names provided by the package so that they are
+# not installed twice in different files and remains idempotent.
+- name: Ensure libselinux-python is installed
+  package:
+    name: libselinux-python
+    state: present
+  register: result
+  until: result is succeeded
+
+- name: Remove openshift_additional.repo file
+  file:
+    dest: /etc/yum.repos.d/openshift_additional.repo
+    state: absent
+
+- name: Create any additional repos that are defined
+  yum_repository:
+    description: "{{ item.description | default(item.name | default(item.id)) }}"
+    name: "{{ item.name | default(item.id) }}"
+    baseurl: "{{ item.baseurl }}"
+    gpgkey: "{{ item.gpgkey | default(omit)}}"
+    gpgcheck: "{{ item.gpgcheck | default(1) }}"
+    sslverify: "{{ item.sslverify | default(1) }}"
+    sslclientkey: "{{ item.sslclientkey | default(omit) }}"
+    sslclientcert: "{{ item.sslclientcert | default(omit) }}"
+    sslcacert: "{{ item.sslcacert | default(omit) }}"
+    file: "{{ item.name }}"
+    enabled: "{{ item.enabled | default('no')}}"
+  with_items: "{{ openshift_additional_repos }}"
+  when:
+  - openshift_additional_repos is defined
+  - openshift_additional_repos | length > 0
+#  notify: refresh cache
+
+- name: Configure origin gpg keys
+  copy:
+    src: "files/openshift-ansible-CentOS-SIG-PaaS"
+    dest: "/etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-SIG-PaaS"
+
+- name: Configure correct origin release repository
```

Comment:
After looking into this further, we don't need these origin repositories at all.  They are all 3.x content anyway.  We are installing clients and hyperkube from `origin-pr` as configured by CI.